_posts/2015-12-08-week-in-review.md

```diff
@@ -0,0 +1,33 @@
+---
+layout:     post
+title:      Slack Highlights
+date:       2015-12-08
+author:     Max Wofford
+summary:    Here's a few highlighted moments from Hack Club's Slack from the past week
+---
+
+_This post is inspired by the format of
+[Hacker News Highlights](https://themacro.com/articles/2015/12/hacker-news-highlights-2/)._
+
+**Here's a few highlighted moments from Hack Club's Slack from the past week.**
+
+--------------------------------------------------------------------------------
+
+A conversation between a club leader and a high school teacher about the utility
+of significant figures.
```

Comment:
I might link the actual people involved. So "a conversation between [hardmath123](https://hardmath123.github.io/) and [mhesby](link)". Saying a "club leader" and "high school teacher" sounds a little impersonal.